Notepad/HomeViewController.swift

```diff
@@ -12,25 +12,38 @@ class HomeViewController: UIViewController {
     @IBOutlet var bodyTextView: UITextView!
     @IBOutlet var titleTextField: UITextField!
     
+    let title_key = "Title"
+    let body_key = "Body"
     override func viewDidLoad() {
         super.viewDidLoad()
     }
     
     @IBAction func handleSaveButtonPressed(_ sender: Any) {
-        let title = titleTextField.text
-        print("my title is: \(title!)")
         
-        let notes = bodyTextView.text
-        print("my body is: \(notes!)")
         
-        UserDefaults.standard.set(title, forKey: "Title")
-        UserDefaults.standard.set(notes, forKey: "Body")
+        let alert = UIAlertController(title: " ", message: "Would you like to save your note?", preferredStyle: .alert)
+        alert.addAction(UIAlertAction(title: "No", style: .cancel, handler: nil))
+    
+        alert.addAction(UIAlertAction(title: "Yes", style: .default, handler: { (UIAlertAction) in
+            if let title = self.titleTextField.text, let body = self.bodyTextView.text {
```

Comment:
To remove unnecessary lines of code, can you try setting `self.titleTextField.text` and `self.bodyTextView.text` directly to `UserDefaults` instead of extracting them out into `title` and `body` variables?